Vite compatibility

```
$ yarn vite
yarn run v1.22.18
$ node ./bin/vite
running: vite --config C:\DEV\test\test\node_modules\vue-cli-plugin-vite\config\index.ts
Pre-bundling dependencies:
  vue
  @aeternity/aepp-sdk
(this will be run only when your dependencies or config have changed)
 > node_modules/@aeternity/aepp-callda
ta/src/utils/Int2ByteArray.js:4:17: er
ror: Big integer literals are not avai
lable in the configured target environ
ment ("chrome87", "edge88", "es2019", "firefox78", "safari13.1")
    4 │     if (bigInt < 256n) {      
      ╵                  ~~~~

 > node_modules/@aeternity/aepp-callda
ta/src/utils/Int2ByteArray.js:9:35: er
ror: Big integer literals are not avai
lable in the configured target environ
ment ("chrome87", "edge88", "es2019", "firefox78", "safari13.1")
    9 │ ...t2ByteArray(bigInt >> 8n),
      ╵                          ~~   

 > node_modules/@aeternity/aepp-callda
ta/src/Serializers/BitsSerializer.js:2
1:54: error: Big integer literals are
not available in the configured target
 environment ("chrome87", "edge88", "es2019", "firefox78", "safari13.1")    
    21 │ ...teTag.POS_BITS ? 1n : -1n
       ╵                     ~~       

 > node_modules/@aeternity/aepp-callda
ta/src/Serializers/TupleSerializer.js:
36:18: error: Big integer literals are
 not available in the configured targe
t environment ("chrome87", "edge88", "es2019", "firefox78", "safari13.1")   
    36 │         let len = 0n
       ╵                   ~~

 > node_modules/@aeternity/aepp-callda
ta/src/Serializers/ListSerializer.js:3
3:18: error: Big integer literals are
not available in the configured target
 environment ("chrome87", "edge88", "es2019", "firefox78", "safari13.1")    
    33 │         let len = 0n
       ╵                   ~~

 > node_modules/@aeternity/aepp-callda
ta/src/Serializers/MapSerializer.js:44
:20: error: Big integer literals are n
ot available in the configured target
environment ("chrome87", "edge88", "es2019", "firefox78", "safari13.1")     
    44 │         if (len === 0n) {    
       ╵                     ~~       

 > node_modules/@aeternity/aepp-callda
ta/src/Serializers/IntSerializer.js:32
:35: error: Big integer literals are n
ot available in the configured target
environment ("chrome87", "edge88", "es2019", "firefox78", "safari13.1")     
    32 │ ...  ...RLPInt(absVal - 64n)
       ╵                         ~~~  

 > node_modules/@aeternity/aepp-callda
ta/src/utils/Int2ByteArray.js:10:24: e
rror: Big integer literals are not ava
ilable in the configured target enviro
nment ("chrome87", "edge88", "es2019", "firefox78", "safari13.1")
    10 │ ...   Number(bigInt & 0xffn)
       ╵                       ~~~~~  

 > node_modules/@aeternity/aepp-callda
ta/src/Serializers/BitsSerializer.js:2
1:60: error: Big integer literals are
not available in the configured target
 environment ("chrome87", "edge88", "es2019", "firefox78", "safari13.1")    
    21 │ ...teTag.POS_BITS ? 1n : -1n
       ╵                           ~~

 > node_modules/@aeternity/aepp-callda
ta/src/Serializers/TupleSerializer.js:
49:19: error: Big integer literals are
 not available in the configured targe
t environment ("chrome87", "edge88", "es2019", "firefox78", "safari13.1")   
    49 │             len += 16n       
       ╵                    ~~~       

 > node_modules/@aeternity/aepp-callda
ta/src/Serializers/ListSerializer.js:3
8:19: error: Big integer literals are
not available in the configured target
 environment ("chrome87", "edge88", "es2019", "firefox78", "safari13.1")    
    38 │             len += 16n       
       ╵                    ~~~       

 > node_modules/@aeternity/aepp-callda
ta/src/Serializers/MapSerializer.js:56
:21: error: Big integer literals are n
ot available in the configured target
environment ("chrome87", "edge88", "es2019", "firefox78", "safari13.1")     
    56 │ ... (let i = 0n; i < len;...
       ╵              ~~

 > node_modules/@aeternity/aepp-callda
ta/src/Serializers/IntSerializer.js:39
:31: error: Big integer literals are n
ot available in the configured target
environment ("chrome87", "edge88", "es2019", "firefox78", "safari13.1")     
    39 │ ...  ...RLPInt(absVal - 64n)
       ╵                         ~~~  

 > node_modules/@aeternity/aepp-callda
ta/src/Serializers/TupleSerializer.js:
59:21: error: Big integer literals are
 not available in the configured targe
t environment ("chrome87", "edge88", "es2019", "firefox78", "safari13.1")   
    59 │ ... (let i = 0n; i < len;...
       ╵              ~~

 > node_modules/@aeternity/aepp-callda
ta/src/Serializers/ListSerializer.js:5
4:21: error: Big integer literals are
not available in the configured target
 environment ("chrome87", "edge88", "es2019", "firefox78", "safari13.1")    
    54 │ ... (let i = 0n; i < len;...
       ╵              ~~

 > node_modules/@aeternity/aepp-callda
ta/src/Serializers/IntSerializer.js:67
:58: error: Big integer literals are n
ot available in the configured target
environment ("chrome87", "edge88", "es2019", "firefox78", "safari13.1")     
    67 │ ...ag.POS_BIG_INT ? 1n : -1n
       ╵                     ~~       

 > node_modules/@aeternity/aepp-callda
ta/src/Serializers/IntSerializer.js:67
:64: error: Big integer literals are n
ot available in the configured target
environment ("chrome87", "edge88", "es2019", "firefox78", "safari13.1")     
    67 │ ...ag.POS_BIG_INT ? 1n : -1n
       ╵                           ~~

 > node_modules/@aeternity/aepp-callda
ta/src/Serializers/ListSerializer.js:5
9:20: error: Big integer literals are
not available in the configured target
 environment ("chrome87", "edge88", "es2019", "firefox78", "safari13.1")    
    59 │         if (len === 0n) {    
       ╵                     ~~       

 > node_modules/@aeternity/aepp-callda
ta/src/Serializers/IntSerializer.js:72
:33: error: Big integer literals are n
ot available in the configured target
environment ("chrome87", "edge88", "es2019", "firefox78", "safari13.1")     
    72 │ ...ateInt((i + 64n) * sign),
       ╵                ~~~

 > node_modules/@aeternity/aepp-callda
ta/src/utils/abs.js:2:34: error: Big i
nteger literals are not available in t
he configured target environment ("chr
ome87", "edge88", "es2019", "firefox78", "safari13.1")
    2 │ ... val > 0 ? val : val * -1n
      ╵                            ~~

error when starting dev server:
Error: Build failed with 20 errors:   
node_modules/@aeternity/aepp-calldata/
src/Serializers/BitsSerializer.js:21:5
4: error: Big integer literals are not
 available in the configured target en
vironment ("chrome87", "edge88", "es2019", "firefox78", "safari13.1")       
node_modules/@aeternity/aepp-calldata/
src/Serializers/BitsSerializer.js:21:6
0: error: Big integer literals are not
 available in the configured target en
vironment ("chrome87", "edge88", "es2019", "firefox78", "safari13.1")       
node_modules/@aeternity/aepp-calldata/
src/Serializers/IntSerializer.js:32:35
: error: Big integer literals are not
available in the configured target env
ironment ("chrome87", "edge88", "es2019", "firefox78", "safari13.1")        
node_modules/@aeternity/aepp-calldata/
src/Serializers/IntSerializer.js:39:31
: error: Big integer literals are not
available in the configured target env
ironment ("chrome87", "edge88", "es2019", "firefox78", "safari13.1")        
node_modules/@aeternity/aepp-calldata/
src/Serializers/IntSerializer.js:67:58
: error: Big integer literals are not
available in the configured target env
ironment ("chrome87", "edge88", "es2019", "firefox78", "safari13.1")        
...
    at failureErrorWithLog (C:\DEV\tes
t\test\node_modules\esbuild\lib\main.js:1493:15)
    at C:\DEV\test\test\node_modules\esbuild\lib\main.js:1151:28
    at runOnEndCallbacks (C:\DEV\test\
test\node_modules\esbuild\lib\main.js:941:63)
    at buildResponseToResult (C:\DEV\t
est\test\node_modules\esbuild\lib\main.js:1149:7)
    at C:\DEV\test\test\node_modules\esbuild\lib\main.js:1258:14
    at C:\DEV\test\test\node_modules\esbuild\lib\main.js:629:9
    at handleIncomingPacket (C:\DEV\te
st\test\node_modules\esbuild\lib\main.js:726:9)
    at Socket.readFromStdout (C:\DEV\t
est\test\node_modules\esbuild\lib\main.js:596:7)
    at Socket.emit (node:events:390:28)
    at addChunk (node:internal/streams/readable:315:12)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs
/cli/run for documentation about this command.

```
We can't replace BigInts, probably it can be adjusted in Vite (https://vitejs.dev/) settings, need to add a possible workaround to https://github.com/aeternity/aepp-sdk-js/blob/develop/docs/compatibility.md

--- a/docs/compatibility.md
+++ b/docs/compatibility.md
@@ -2,23 +2,25 @@
 
 This package is expected to work in these environments:
 
-| Environment                           | Comment                                                              |
-|---------------------------------------|----------------------------------------------------------------------|
-| nodejs>=14.19, common js              |                                                                      |
-| nodejs>=14.19, mjs                    |                                                                      |
-| Browser using script tag, umd         |                                                                      |
-| webpack@4                             | requires a fix to work with mjs build [webpack-4]                    |
-| webpack@5                             |                                                                      |
-| @vue/cli@4 (webpack@4)                |                                                                      |
-| @vue/cli@5 (webpack@5)                |                                                                      |
-| create-react-app@4 (webpack@4)        | mjs build is not compatible with webpack@4 [cra-webpack-4]           |
-| create-react-app@5 (webpack@5)        |                                                                      |
-| create-react-native-app@3 (webpack@4) | mjs build is not compatible with webpack@4 [cra-webpack-4]           |
-| meteor@2                              |                                                                      |
-| jest@27.5.1                           | requires an environment where Buffer is instanceof Uint8Array [jest] |
-| typescript                            | requires `tsconfig.json` adjustments [typescript]                    |
+| Environment                           | Comment                                                                       |
+|---------------------------------------|-------------------------------------------------------------------------------|
+| nodejs>=14.19, common js              |                                                                               |
+| nodejs>=14.19, mjs                    |                                                                               |
+| Browser using script tag, umd         |                                                                               |
+| webpack@4                             | requires a fix to work with mjs build [webpack-4]                             |
+| webpack@5                             |                                                                               |
+| @vue/cli@4 (webpack@4)                |                                                                               |
+| @vue/cli@5 (webpack@5)                |                                                                               |
+| create-react-app@4 (webpack@4)        | mjs build is not compatible with webpack@4 [cra-webpack-4]                    |
+| create-react-app@5 (webpack@5)        |                                                                               |
+| create-react-native-app@3 (webpack@4) | mjs build is not compatible with webpack@4 [cra-webpack-4]                    |
+| meteor@2                              |                                                                               |
+| jest@27.5.1                           | requires an environment where Buffer is instanceof Uint8Array [jest]          |
+| typescript                            | requires `tsconfig.json` adjustments [typescript]                             |
+| vite@3                                | requires `build.target: 'es2020'` and `bigint: true` in vite.config.js [vite] |
 
 [webpack-4]: https://github.com/webpack/webpack/issues/7482#issuecomment-394884837
 [cra-webpack-4]: https://github.com/aeternity/aepp-sdk-js/issues/1529
 [jest]: https://github.com/facebook/jest/issues/4422#issuecomment-770274099
 [typescript]: index.md#typescript-projects
+[vite]: https://github.com/vitejs/vite/issues/9062#issuecomment-1202167352

